rearranging (#2718)
ROCm/ROCm: RELEASE.md

# Release notes for AMD ROCm™ 6.0

ROCm 6.0 is a major release with new performance optimizations, expanded frameworks and library
support, and improved developer experience. This includes initial enablement of the AMD Instinct™
MI300 series. Future releases will further enable and optimize this new platform. Key features include:

* Improved performance in areas like lower precision math and attention layers.
* New hipSPARSELt library to accelerate AI workloads via AMD's sparse matrix core technique.
* Latest upstream support for popular AI frameworks like PyTorch, TensorFlow, and JAX.
* New support for libraries, such as DeepSpeed, ONNX-RT, and CuPy.
* Prepackaged HPC and AI containers on AMD Infinity Hub, with improved documentation and
  tutorials on the [AMD ROCm Docs](https://rocm.docs.amd.com) site.
* Consolidated developer resources and training on the new AMD ROCm Developer Hub.

The following section provide a release overview for ROCm 6.0. For additional details, you can refer to
the [Changelog](https://rocm.docs.amd.com/en/develop/about/CHANGELOG.html).

## OS and GPU support changes

> AMD Instinct™ MI300A and MI300X Accelerator support has been enabled for limited operating

Reviewer: ```suggestion
ROCm 6.0 enables the use of MI300A and MI300X Accelerators with a limited operating systems support. 
Future releases will add additional OS's to match our general offering. ```

> systems.

Reviewer: ```suggestion
```

> * Ubuntu 22.04.5 (MI300A and MI300X)
> * RHEL 8.9 (MI300A)
> * SLES 15 SP5

Reviewer: ```suggestion
| Operating Systems | MI300A | MI300X |
|:---:|:---:|:---:|
| Ubuntu 22.04.5 | Supported | Supported |
| RHEL 8.9 | Supported |  |
| SLES15 SP5 | Supported |  |
```

> We've added support for the following operating systems:

Reviewer: ```suggestion
For older generations of supported Instinct products we have added the following operating systems:
```

* RHEL 9.3
* RHEL 8.9

> Note that, of ROCm 6.2, we've planned for end-of-support (EoS) for the following operating systems:

Reviewer: ```suggestion
Note: For ROCm 6.2 and beyond, we've planned for end-of-support (EoS) for the following operating systems:
```
